src/lib/coda_networking/coda_networking.ml

```diff
@@ -311,11 +308,9 @@ module type Inputs_intf = sig
 
   module Ledger_hash : Protocols.Coda_pow.Ledger_hash_intf
 
-  module Staged_ledger_hash :
-    Protocols.Coda_pow.Staged_ledger_hash_intf
-    with type staged_ledger_aux_hash := Staged_ledger_aux_hash.t
-     and type ledger_hash := Ledger_hash.t
-
+  (* we omit Staged_ledger_hash, because the available module in Inputs is not versioned; instead, in the
+     versioned RPC modules, we use a specific version
+   *)
```

Comment:
no need for this comment -- even if we didn't need to use the specific version. It's always nice to remove an unnecessary functor argument.

Reply:
The Staged_ledger_hash is contained in `Inputs`, which is passed to several functors. We're deliberately not using it here, so the comment clarifies that design decision. Maybe it's useful to mention that, in case someone wonders why we're not using that argument.